docs/releases/1.11.5.txt

```diff
@@ -4,7 +4,13 @@ Django 1.11.5 release notes
 
 *Under development*
 
-Django 1.11.5 fixes several bugs in 1.11.4.
+Django 1.11.5 adds a minor feature and fixes several bugs in 1.11.4.
```

Comment:
Per our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions), features are not backported to the stable branch. This should target Django 2.0 instead.